api-module-library/hubspot/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@friggframework/api-module-hubspot",
-    "version": "0.9.1",
+    "version": "0.10.0",
```

Comment:
This PR contains breaking changes, the version should be bumped to `1.0.0` at least. Still, a question mark if this would be ready for a v1.